DiskIOMeter.c

```diff
@@ -56,21 +57,26 @@ static void DiskIOMeter_updateValues(Meter* this, char* buffer, size_t len) {
       }
 
       if (data.totalBytesRead > cached_read_total) {
-         cached_read_diff = (data.totalBytesRead - cached_read_total) / 1024; /* Meter_humanUnit() expects unit in kilo */
+         diff = data.totalBytesRead - cached_read_total;
+         diff /= 1024; /* Meter_humanUnit() expects unit in kilo */
+         cached_read_diff = (unsigned int)diff;
       } else {
-         cached_read_diff = 0;
+         cached_read_diff = 0UL;
       }
       cached_read_total = data.totalBytesRead;
 
       if (data.totalBytesWritten > cached_write_total) {
-         cached_write_diff = (data.totalBytesWritten - cached_write_total) / 1024; /* Meter_humanUnit() expects unit in kilo */
+         diff = data.totalBytesWritten - cached_write_total;
+         diff /= 1024; /* Meter_humanUnit() expects unit in kilo */
+         cached_write_diff = (unsigned int)diff;
```

Comment:
Type truncation: `unsigned int` casted, but `cached_write_diff` is `unsigned long int`…